Week1/homework/app.js

```diff
@@ -1,11 +1,187 @@
-'use strict';
+/* eslint-disable no-unused-expressions */
```

Comment:
`eslint-disable`? That is cheating haha, since it's not in the homework for week 1 its fine :)

Reply:
haha, No cheating! :) but I think my Eslint has technical problem, It gave me some messages and I treid to solve it but it did not work! I will try on sunday also to understand what is happen.